spec.md

```diff
@@ -814,7 +815,7 @@ The request provides these query string parameters as useful hints for brokers.
 ##### cURL
 ```
 $ curl 'http://username:password@broker-url/v2/service_instances/:instance_id?accepts_incomplete=true
-  &service_id=service-id-here&plan_id=plan-id-here' -X DELETE -H "X-Broker-API-Version: 2.11"
+  &service_id=service-id-here&plan_id=plan-id-here' -X DELETE -H "X-Broker-API-Version: 2.12"
```

Comment:
@pmorie I see what you mean now, but I'm not going to add that to the release proposal as it is an existing issue with the spec. We can fix that in a future PR (fancy doing that one? :) )